syncer/syncer.go

```diff
@@ -293,7 +296,7 @@ func (s *syncer) bulkSyncWithPeer(peerID peer.ID, newBlockCallback func(*types.B
 				return lastReceivedNumber, false, fmt.Errorf("unable to verify block, %w", err)
 			}
 
-			if err := s.blockchain.WriteBlock(block); err != nil {
+			if err := s.blockchain.WriteBlock(block, "syncer"); err != nil {
```

Comment:
Why not define constant?

Reply:
Defined:

[24b99b1](https://github.com/0xPolygon/polygon-edge/pull/650/commits/24b99b1298eb81b13e66aa46bfb57faa6d386d60)